doc/source/notebooks/intro_to_gpflow2.pct.py

```diff
@@ -477,3 +477,31 @@ def checkpointing_training_loop(
 loaded_result = loaded_model.predict(samples_input)
 
 np.testing.assert_array_equal(loaded_result, original_result)
+
+# %% [markdown]
+# ## User config update
+#
+# In this notebook, we used a lot `gpflow.config` methods for setting and getting default attributes from global configuration. However, GPflow provides a way for local config modification without updating values in global. As you can see below, using `gpflow.config.as_context` replaces temporarily global config with your instance. At creation time, custom config instance uses standard values from the global config:
+
+# %%
+user_config = gpflow.config.Config(float=tf.float32, positive_bijector="exp")
+
+user_str = "User config\t"
+global_str = "Global config\t"
```

Comment:
Nit: I got a bit confused why we'd need a \tab in here - I might have moved that into the print() call itself...

Reply:
I didn't want to pollute prints with tab symbol